lib/src/config/error.rs

```diff
@@ -48,6 +48,12 @@ pub enum ConfigError {
     ///
     /// Parameters: (key)
     UnknownKey(String),
+    /// The address is unsupported on this platform
+    UnsupportedAddress,
+    /// The unix socket path already exists
+    ///
+    /// Parameters: (path)
+    BadUnixSocketPath(PathBuf),
```

Comment:
I am not sure about the naming for `UnsupportedAddress` and `BadUnixSocketPath`.

Reply:
This is good.